js/impress.js

```diff
@@ -397,6 +397,33 @@
                 scale:     1
             };
             
+            // Hide the mouse cursor in fullscreenmode after the configured
```

Comment:
We are moving to a plugin architecture. So instead of a patch against impress.js, you should write this as a plugin. See https://github.com/henrikingo/impress.js/blob/myfork/src/plugins/README.md for a detailed explanation. If you want to write a PR against that repo, you're welcome to do so.

Alternatively, if you want to do a PR against the impress/impress.js repo in its current state, you could still write this as a "plugin" By adding a separate `(function(window, document){...})(window, document);` at the end of the file. You can take a queue from the keypress navigation at the end of the file, which is in fact such a plugin (even if it's not in a separate file).